books/RayTracingInOneWeekend.html

```diff
@@ -386,19 +391,23 @@
 --------------
 <div class='together'>
 The one thing that all ray tracers have is a ray class, and a computation of what color is seen
-along a ray. Let’s think of a ray as a function $\mathbf{p}(t) = \mathbf{a} + t \vec{\mathbf{b}}$.
-Here $\mathbf{p}$ is a 3D position along a line in 3D. $\mathbf{a}$ is the ray origin and
+along a ray. Let’s think of a ray as a function:
+
+  $$ \mathbf{p}(t) = \mathbf{a} + t \vec{\mathbf{b}} $$
+
+Where $\mathbf{p}$ is a 3D position along a line in 3D, $\mathbf{a}$ is the ray origin, and
 $\vec{\mathbf{b}}$ is the ray direction. The ray parameter $t$ is a real number (`double` in the
-code). Plug in a different $t$ and $p(t)$ moves the point along the ray. Add in negative $t$ and you
-can go anywhere on the 3D line. For positive $t$, you get only the parts in front of $\mathbf{a}$,
-and this is what is often called a half-line or ray.
+code). Plug in a different $t$ and $\mathbf{p}(t)$ moves the point along the ray. Add in negative
+$t$ and you can go anywhere on the 3D line. For positive $t$, you get only the parts in front of
+$\mathbf{a}$, and this is what is often called a half-line or ray. This is where the _ray_ in
+_ray tracing_ comes from.
```

Comment:
Please revert; this collides with my in-progress math revisions. Another reason I don't like grab-bag PRs.

Reply:
Also, points are uppercase bold, vectors are lowercase bold.

Reply:
thoughts on "this is where the ray in tray tracing comes from"? Cut?

Reply:
Hmm. I don't think it needs to be said.

Reply:
Cutting